beacon-chain/state/stateutil/field_root_validator.go

```diff
@@ -51,31 +54,53 @@ func validatorRegistryRoot(validators []*ethpb.Validator) ([32]byte, error) {
 	return res, nil
 }
 
+func hashValidatorHelper(validators []*ethpb.Validator, roots [][32]byte, j int, groupSize int, wg *sync.WaitGroup) {
+	defer wg.Done()
+	for i := 0; i < groupSize; i++ {
+		fRoots, err := ValidatorFieldRoots(validators[j*groupSize+i])
+		if err != nil {
+			logrus.WithError(err).Error("could not get validator field roots")
```

Comment:
I don't understand why happens if you error here, `fRoots` will be junk right?

Reply:
yep

Reply:
Then the whole function is useless. Why not `continue` or `return`?